soc/arm/nuvoton_npcx/npcx7/Kconfig.soc

```diff
@@ -33,3 +33,23 @@ config SOC_POWER_MANAGEMENT_TRACE
 	depends on SOC_POWER_MANAGEMENT
 	help
 	  Internal config to enable runtime power management traces.
+
+config SOC_POWER_CONSOLE_EXPIRED
+	bool "Enable support for console expired mechanism"
+	default y
+	depends on PM
+	help
+	  Enable support for console expired mechanism. It adds the notification
+	  to the power management module that the module for console is in use.
+	  If the interval that module doesn't receive any input message exceeds
+	  CONFIG_SOC_POWER_CONSOLE_EXPIRED_TIMEOUT, the power management module
+	  is allowed to enter deep sleep mode. This mechanism gives a window in
+	  which the users can organize console input.
+
+config SOC_POWER_CONSOLE_EXPIRED_TIMEOUT
+	int "Fixed amount of time to keep the console in use flag true"
+	default 15000
+	depends on SOC_POWER_CONSOLE_EXPIRED
+	help
+	  Fixed amount of time which unit is milli-second to keep the console in
+	  use flag true.
```

Comment:
Can these configs get promoted to work with the general CONFIG_UART_CONSOLE?

Reply:
Do you mean that?
1. Replace the prefix "SOC_POWER_CONSOLE_" with "UART_CONSOLE_".
2. Move these two configs to [console driver's Kconfig file](https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/console/Kconfig#L47)

If so, I will modify it later.

Reply:
Done.